internal/repository/instance/smtp_config.go

```diff
@@ -10,16 +10,19 @@ import (
 
 const (
 	smtpConfigPrefix                   = "smtp.config."
+	httpConfigPrefix                   = "http."
```

Comment:
shouldn't we have a email.config with smtp and http specific events?
or do we want to keep it for the moment / not break right now?